package.json

```diff
@@ -40,6 +44,7 @@
     "@webcomponents/webcomponentsjs": "^2.0.0",
     "babel-eslint": "^10.0.1",
     "chai": "^4.2.0",
+    "check-engine": "1.8.1",
```

Comment:
I would not worry too much about locking this so it works with node 6, which is has been dead for a long time. Specially since we will stop using it when 9.10 reaches EOF, which is just around the corner.